docker-compose/assets/opencast-ddl.sql

```diff
@@ -1,5 +1,3 @@
--- Created with Opencast version next
-
```

Comment:
Could you add this comment? There is a mechanism to generate the DDL script from a given Opencast image (see README.md) and this comment is always included to indicate the version.

Reply:
If its creating the DDL script given the opencast image is not working properly, I have to manually set the DDL from opencast version 7 because the original script have some tables missing

Reply:
The image does not automatically use the DDL script (well Opencast can). What I mean is that given an Opencast image Docker you can run this command
```sh
$ docker run -it --rm \
  -e ORG_OPENCASTPROJECT_DB_VENDOR=MySQL \
  quay.io/opencast/allinone:next \
  app:print:ddl
```
to print out the DDL script necessary for this Opencast version to work. The output will include the above header to indicate which version of Opencast this script is tight to. Therefore I was asking to add it again.

Reply:
Ok, now I understand better. It was for me very strange that comment because says "Created" like generated from a Script.

So is not necessary when you use Compose but you need that comment when you run as a standalone image. :) Have I understood correctly?